paasta_tools/cli/cmds/spark_run.py

```diff
@@ -439,7 +439,10 @@ def add_subparser(subparsers):
 
     aws_group.add_argument(
         "--assume-aws-role",
-        help="Takes an AWS IAM role ARN and attempts to create a session",
+        help=(
+            "Takes an AWS IAM role ARN and attempts to create a session using "
+            "spark_role_assumer"
```

Comment:
I think would be useful to add a one line for what using "spark_role_assumer" mean for someone using the docs to understand this parameter.